generators/app/index.js

```diff
@@ -103,6 +103,13 @@ module.exports = class extends BaseGenerator {
             type: String
         });
 
+        // This adds support for a `--cache-impl` flag
```

Comment:
I dont see a need to provide this via flags, the flags are only for special case handlings, so please remove this or provide a reasonable explanation on why this is needed